docs/conf.py

```diff
@@ -91,6 +91,13 @@ def render_hubs():
         yaml = cluster_info.read_text()
         cluster = safe_load(yaml)
 
+        # Pull support chart information to populate fields (if it exists)
+        support = cluster.get("support", {}).get("config", {})
+        grafana_url = support.get("grafana", {}).get("ingress", {}).get("hosts", "")
+        if isinstance(grafana_url, list):
```

Comment:
Are we really expecting a list of hosts urls? And if that is the case, why do we choose just the first one?

Reply:
Yeah, the `grafana` chart expects `ingress.hosts` to be a list:

https://github.com/grafana/helm-charts/blob/a0b7d4d2222976bd11af38b2fec3f654a764030a/charts/grafana/values.yaml#L180-L196

I think if we are following the same policy for grafana that we do for hubs, that we don't support multiple domains, then there *should* only be one entry here. Maybe that is something we can enforce in the schema?

Reply:
yeah I was just taking the first item because it always seemed to be a list in the config. Is this something that should block this PR? I think building the docs will fail if this condition is ever not met so we'd catch it thne

Reply:
Should we capture @sgibson91's suggestion in a new issue?

Reply:
+1 from me